code/modules/surgery/organs/ears.dm

```diff
@@ -106,11 +106,62 @@
 	damage_multiplier = 0.9
 	organ_flags = ORGAN_SYNTHETIC
 
-/obj/item/organ/internal/ears/cybernetic/upgraded
-	name = "upgraded cybernetic ears"
+/obj/item/organ/internal/ears/cybernetic/translation
+	name = "cybernetic translation ears"
 	icon_state = "ears-c-u"
-	desc = "An advanced cybernetic ear, surpassing the performance of organic ears."
+	desc = "Allows the user to understand one of several commonly spoken languages. Grants no ability to speak these languages."
 	damage_multiplier = 0.5
+	var/list/language_list = list(/datum/language/common, /datum/language/uncommon, /datum/language/moffic, /datum/language/draconic, /datum/language/calcic, /datum/language/voltaic, /datum/language/nekomimetic)
+	// It needs to cycle through all of the languages in the list. This seems like the most elegant solution.
+	var/language_selector = 1
+	// Byond doesn't like it when I set it equal to language_list[language_selector], so this will have to do
+	var/datum/language/chosen_language = /datum/language/common
+
+/obj/item/organ/internal/ears/cybernetic/translation/examine(mob/user)
+	. = ..()
+	. += span_info("[src] are currently translating [initial(chosen_language.name)] speech. You can use a screwdriver to change the language.")
+
+/obj/item/organ/internal/ears/cybernetic/translation/screwdriver_act(mob/living/user, obj/item/screwtool)
+	. = ..()
+	if(.)
+		return TRUE
+
+	// language_selector += 1
+	// if (language_selector > language_list.len)
+	// 	language_selector = 1
+
+	// Should be the same output as above, but a little more compact
+	language_selector = (language_selector % language_list.len) + 1
+	chosen_language = language_list[language_selector]
+
+	to_chat(user, span_notice("You modify [src] to translate [initial(chosen_language.name)] speech."))
```

Comment:
This is really terrible UX. Can we have the screwdriver open up an input list instead?

Reply:
I have changed this to an input list in the latest commit.